xswap/__init__.py

```diff
@@ -0,0 +1 @@
+from .xswap import permute_edge_list  # noqa: F401
```

Comment:
see whether it's appropriate to use __all__ rather than noqa to suppress flakelint